test/db/tools/rz_find

```diff
@@ -260,3 +260,10 @@ EXPECT=<<EOF
 EOF
 RUN
 
+NAME=rz-find -E with echo as a command
+FILE==
+CMDS=!rz-find -s "README" -E "echo" bins/arm~README~?
+EXPECT=<<EOF
+0
```

Comment:
That test is bad, to be honest. Why zero? It should have one match

Reply:
It's actually Incorrect. The command `!rz-find -s "README" -E "echo" bins/arm` correctly finds the addresses of 'README'. However, when I filter these results with `~README~` and count lines with `?`, it returns `0`. Despite 'README' being found, the filter seems to fail.

Reply:
I think it's because we don't have support for nested greps yet: https://github.com/rizinorg/rizin/issues/904
Please try to make a test with just one internal grep

Reply:
Okay Sure.

Reply:
@wargio you can see this